src/test/addresses_goerli.sol

```diff
@@ -350,6 +350,8 @@ contract Addresses {
         addr["STARKNET_DAI_BRIDGE_LEGACY"]      = 0xd8beAa22894Cd33F24075459cFba287a10a104E4;
         addr["STARKNET_GOV_RELAY"]              = 0x73c0049Dd6560E644984Fa3Af30A55a02a7D81fB;
         addr["STARKNET_CORE"]                   = 0xde29d060D45901Fb19ED6C6e959EB22d8626708e;
+        addr["STARKNET_TELEPORT_BRIDGE"]        = 0x140f746CcFbb1C2618838C063048949685d7A6eD;
```

Comment:
Probably better to insert these before "STARKNET_DAI_BRIDGE", to maintain the same order as in optimism and arbitrum.